api/services/EmailTemplates/forgottenUsername.ejs

```diff
@@ -0,0 +1,3 @@
+<p>
+  Your username for your account at <a href="http://<%= siteUrl %>"><%= siteUrl %></a> is <%= recipientName %>.
```

Comment:
I guess the best way to do this, then, is to just send an email with all of the usernames?